travis/git-crypt-setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+
+if [ "$#" -ne 0 ]; then
+    echo "Add Travis CI to a git-crypt set up for a repository"
+	echo "Will create an uncommitted diff."
+    echo "Usage: $0"
+	exit 1
+fi
+
+email="travis-ci@example.com"
```

Comment:
Can this be moved to line 27 or close to its first usage?

Reply:
Should be no problem, it can't be overridden (yet).